index.js

```diff
@@ -22,13 +22,13 @@ function plugin (options) {
         helperContents = require(path)
 
         switch (typeof helperContents) {
-        case 'function':
-          templateName = file.split('.').shift()
-          Handlebars.registerHelper(templateName, helperContents)
-          break
-        case 'object':
-          Handlebars.registerHelper(helperContents)
-          break
+          case 'function':
```

Comment:
You have forced whitespace changes 😗

Reply:
Oh, thanks!

Reply:
@mischah i checked it (https://github.com/nicolaisueper/metalsmith-register-helpers/blob/aff435942104dc443fdc6a68741d6eb5402d22ec/index.js) and i changed it according to the 'standard' code style applied on this project. This should be right.

Reply:
@kennethormandy please check this :)

Reply:
ah. Sorry. 👌